src/components/input-dropdown/input-dropdown.component.spec.ts

```diff
@@ -46,6 +46,19 @@ describe('InputDropdownComponent', () => {
         expect(component.filterInputElement.nativeElement.focus).toHaveBeenCalledWith();
     });
 
+    it('should focus the input field when dropdownOpen is opened with a click', async () => {
+        spyOn(component.filterInputElement.nativeElement, 'focus');
+
+        let trigger = document.querySelector('button.form-control') as HTMLButtonElement;
+
+        trigger.click();
+
+        fixture.detectChanges();
+        await fixture.whenStable();
+
+        expect(component.filterInputElement.nativeElement.focus).toHaveBeenCalled();
```

Comment:
This should be `expect(document.activeElement).toBe(input);` I think (the opposite of your check in the other test).